### PR TITLE
add langauge name property to ScoreEvent for plugin uses

### PR DESCRIPTION
**Pull request by shajyhia**

### CODE REVIEW

File: score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java [hunk 1]
```diff
@@ -61,6 +61,8 @@
 
     private static final String REQUESTED_EXECUTION_PLAN_ID = "REQUESTED_EXECUTION_PLAN_ID";
 
+    public static final String LANGUAGE_TYPE = "LANGUAGE_TYPE";
+
     protected Map<String, Serializable> contextMap = new HashMap<>();
```
Comment (orius123) [inline score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java]: how do a user set the language name?

Comment (orius123) [inline score-api/src/main/java/io/cloudslang/score/lang/ExecutionRuntimeServices.java]: and no. the answer is not through adding it directly to the system context.

**Comment by shajyhia** [comment]
what is your suggestion ??

**Comment by shajyhia** [comment]
be free to add at merge time

**Pull request merged**